gui/tasks/forms.py

```diff
@@ -186,6 +187,8 @@ def check_rpath_exists(self):
         exists or not. Returns TRUE rpath is a directory
         and exists, else FALSE"""
 
+        self.ssh_host_keyscan()
```

Comment:
Like I said, you must use ssh command line options for the first check.

Also, I'm not sure its secure to simply add the host key without user acknowledge. I guess this deserve some discussion.